packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -22,7 +22,8 @@ describe('common/sqlite', () => {
   test('basic introspection', async () => {
     ctx.fixture('introspection/sqlite')
     const introspect = new DbPull()
-    await introspect.parse(['--print'])
+    const result = introspect.parse(['--print'])
+    await expect(result).resolves.toMatchInlineSnapshot(``)
```

Comment:
What does this test? No stdout? No error? What exactly is the return of this?

Reply:
Yeah that was a more general question - as you are repeating this code in every test - where and when is this _supposed_ return anything?

Reply:
Since we always return an empty string for `.parse()` for `db pull` it will be like this everywhere 
https://github.com/prisma/prisma/blob/75ec7c42ed0825e108bfe98eec6e977ece581a61/packages/migrate/src/commands/DbPull.ts#L317:L317